src/olympia/addons/tests/test_indexers.py

```diff
@@ -102,6 +117,43 @@ def test_extract_attributes(self):
         for field_name in self.simple_fields:
             assert extracted[field_name] == getattr(self.addon, field_name)
 
+        assert extracted['app'] == [FIREFOX.id]
```

Comment:
Probably unrelated to this specific test but why is `app` a list, shouldn't we call this `apps` then?